storybook/src/components/tabs/tabs.css-template.ts

```diff
@@ -8,7 +8,7 @@ export const cssTabs: ComponentImplementation<Tabs<TemplateResult>> = {
   component: "tabs",
   implementation: "html-css",
   template: () =>
-    function tabsTemplate({ items }) {
+    function tabsTemplate({ items, content }) {
       return html`
         <ul class="nav nav-tabs" role="tablist">
           ${items.map(
```

Comment:
In de core template object spread je `item`, zullen we dat in de css template ook doen?

Reply:
Ivm met de deprecation niet doen?

Reply:
Ahja goed punt